cache_test.go

```diff
@@ -806,6 +806,16 @@ func Test_Cache_Range(t *testing.T) {
 	})
 
 	assert.Equal(t, []string{"5", "4"}, results)
+
+	defer func() {
+		r := recover()
+		assert.Equal(t, nil, r, "range on empty cache caused panic")
```

Comment:
This would be a bit clearer if [assert.NotPanics](https://pkg.go.dev/github.com/stretchr/testify/assert#NotPanics) were used.

Reply:
> This would be a bit clearer if [assert.NotPanics](https://pkg.go.dev/github.com/stretchr/testify/assert#NotPanics) were used.

Added that. I didn't new this existed :O